src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -3,80 +3,99 @@ title: 'Environment variables'
+By default, running the above script locally will execute a single iteration using one virtual user(VU). We can modify the **default behavior** by passing along [k6 Options](/using-k6/options) as environment variables:
 
 <CodeGroup labels={["Bash", "Windows: CMD", "Windows: PowerShell"]} lineNumbers={[false]}>
 
 ```bash
-$ MY_HOSTNAME=test.k6.io k6 run script.js
+$ K6_VUS=10 K6_DURATION=10s k6 run script.js
 ```
 
 ```bash
-C:\k6> set "MY_HOSTNAME=test.k6.io" && k6 run script.js
+C:\k6> set "K6_VUS=10 K6_DURATION=10s" && k6 run script.js
 ```
 
 ```bash
-PS C:\k6> $env:MY_HOSTNAME="test.k6.io"; k6 run script.js
+PS C:\k6> $env:K6_VUS=10 ; $env:K6_DURATION="10s" ; k6 run script.js
 ```
 
 </CodeGroup>
 
-or using an [`-e` / `--env` CLI flag](/using-k6/options#supply-environment-variables) (which will be the same for all platforms):
+The same script will now run 10 virtual users for a duration of 10 seconds, resulting in about 70 iteration loops. Take note you must prefix `K6_` in the environment variable name in order for k6 to evaluate it as a **option parameters**.
 
-<CodeGroup labels={[]} lineNumbers={[true]}>
+## 2. Passing System Environment Variables to Script Code
+
+We can also embed environment variables within script code using a global `__ENV` variable, a JS object. The source of the environment variable can be exposed either via system environment variable or via the `e` flag which we'll discuss later in the next section.
+
+Consider the following script example:
+
+```javascript
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function () {
+  const res = http.get(`http://${__ENV.MY_HOSTNAME}/`);
+  sleep(1);
+}
+```
+
+In order to run the above script, we'll need to supply the source value of `__ENV.MY_HOSTNAME` variable as follows:
+
+<CodeGroup labels={["Bash", "Windows: CMD", "Windows: PowerShell"]} lineNumbers={[false]}>
 
 ```bash
-$ k6 run -e MY_HOSTNAME=test.k6.io script.js
+$ MY_HOSTNAME=test.k6.io k6 run script.js
+```
+
+```bash
+C:\k6> set "MY_HOSTNAME=test.k6.io" && k6 run script.js
+```
+
+```bash
+PS C:\k6> $env:MY_HOSTNAME="test.k6.io"; k6 run script.js
 ```
 
 </CodeGroup>
 
-Note: This can *not* be used to configure k6 with environment variables as listed on the [options](/using-k6/options) page. In other words `-e K6_ITERATIONS=120` will *not* configure the script [iterations](/using-k6/options#iterations), it will just provide `__ENV.K6_ITERATIONS` to the script, unlike `K6_ITERATIONS=120 k6 run script.js`.
+Take note that by default, passing system environment variables does not work for `k6 archive`, `k6 cloud` and `k6 inspect`. In order for it to work, you'll need to explicitly specify `--include-system-env-vars`. You can also disable the default passing of system environment variables when running scripts by using `--include-system-env-vars=false`.
 
-The environment variable could then be used as follows in a script:
+## 3. Using the `-e` flag to pass environment variables
```

Comment:
After talking with k6 devs, we want this option to be described as the default/main way to pass environment variables. 

Consider the `2. Passing System Environment Variables to Script Code` to be an alternative or additional FYI subsection.

Reply:
I've rearranged the sections. Will need clarification